lib/events.js

```diff
@@ -394,6 +394,27 @@ EventEmitter.prototype.listeners = function listeners(type) {
   return ret;
 };
 
+EventEmitter.prototype.listenerCount = function listenerCount(type) {
+
+  if (typeof type !== 'string') {
+    throw new TypeError('type must be a string');
+  }
+
+  const events = this._events;
+
+  if (events) {
+    const evlistener = events[type];
+
+    if (typeof evlistener === 'function') {
+      return 1;
+    } else if (evlistener) {
+      return evlistener.length;
+    }
+  }
+
+  return 0;
+};
```

Comment:
Missed that. Should we be checking type? It's a slight change that deviates from previous impl, and so we can't unequivocally say it's possible to just switch usage. I don't care either way. If the check stays in the yeah, can leave it just as is.